sdk/clientcore/core/src/main/java/module-info.java

```diff
@@ -21,6 +21,7 @@
     exports io.clientcore.core.util.configuration;
     exports io.clientcore.core.util.serializer;
     exports io.clientcore.core.util.auth;
+    exports io.clientcore.core.observability;
```

Comment:
Should we put all logging, tracing, etc into this package? Right now, logging is in a utility package and this seems to be a better location.

Reply:
let's do it in the follow up